std/typecons.d

```diff
@@ -6736,6 +6736,31 @@ struct Typedef(T, T init = T.init, string cookie=null)
             TD im() {return TD(Typedef_payload.im);}
         }
     }
+
+    /**
+     * Params:
+     *     writer = A `char` accepting
+     *     $(REF_ALTTEXT output range, isOutputRange, std, range, primitives)
+     *     fmt = A $(REF FormatSpec, std,format) which is used to represent
+     *     the value
+     * Returns:
+     *     A `string` if `writer` and `fmt` are not set; `void` otherwise.
```

Comment:
This is incorrect, as each overload will show up separately in the documentation, and it will confuse readers if you say that this one could return `void` _or_ `string`.

Reply:
(This is copy and paste from Nullable, so its documentation is weak too)